app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
Also, as a style thing, you could make this a bit nicer by replicating what you've already done with your `with_users` variable.

Something like:

```py
only_buyers = request.args.get('only_buyers') == 'true'
```